Source/Components/ImageGlass.UI/packages.config

```diff
@@ -1,4 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="DotNetZip" version="1.13.5" targetFramework="net471" />
+  <package id="Magick.NET-Q16-AnyCPU" version="7.15.0" targetFramework="net471" />
```

Comment:
I prefer not to directly use `Magick.NET` in the project other than `ImageGlass.Heart`. It may cause troublesome when updating the new version.